test/conformance/conformance_test.go

```diff
@@ -50,7 +50,6 @@ func TestGatewayAPIConformance(t *testing.T) {
 		tests.HTTPRouteInvalidNonExistentBackendRef,
 		tests.HTTPRouteInvalidBackendRefUnknownKind,
 		tests.HTTPRouteInvalidCrossNamespaceBackendRef,
-		tests.HTTPRouteHeaderMatching,
```

Comment:
why is this removed ?

Reply:
Looks like this is duplicated. I did not notice that you already added this.